priv/static/phoenix_live_reload.js

```diff
@@ -53,6 +55,27 @@ socket.connect();
 var chan = socket.channel('phoenix:live_reload', {})
 chan.on('assets_change', function(msg) {
   var reloadStrategy = reloadStrategies[msg.asset_type] || reloadStrategies.page;
+
+  if (restoreScrollOnReload && reloadStrategy === pageStrategy) {
```

Comment:
Added a guard here so that css (non-reload) refreshes wouldn't result in scrollY values being saved, as this could in result in a developer making a CSS change, and on the next _deliberate_ reload, having that scroll position restored.

Reply:
Inner monologue: "I should just house this _within_ the pageReload strategy to avoid the extra check here"

(Will update)